app/controllers/user.js

```diff
@@ -1,20 +1,33 @@
 const { registerUser, findUserByEmail } = require('../services/user');
-const { verifyEmail, verifyPassword, encriptPassword } = require('../helpers/user');
+const { encriptPassword, createToken, validatePassword } = require('../helpers/user');
 const logger = require('../logger');
-const { userError } = require('../errors');
-const { USER_ERROR } = require('../../config/messageError');
 
 exports.createUser = async (req, res, next) => {
   try {
+    logger.info(`createUser controller start, request: ${req.body}`);
```

Comment:
Well done but here are two details, the password should not go in the logs and the other is that in this way the console shows something like `[object Object]`.